libkirk/__init__.py

```diff
@@ -13,7 +13,7 @@
 
 
 # Kirk version
-VERSION = '1.1'
+__version__ = '1.1'
```

Comment:
@acerv Why we don't bump version on kirk release? Shouldn't this be `1.3`?

Reply:
Yes, it should be increased indeed. This is something that eventually will be shown in the kirk package in the future, so it's important to fix this...

Reply:
Maybe merge this PR and update version in a separate commit yourself?

Reply:
I added a patch to update the __version__ hope it helps.